docs/src/cli/deploy-a-program.md

```diff
@@ -89,10 +89,10 @@ Data Length: 5216 (0x1460) bytes
 ### Redeploy a program
```

Comment:
We also probably want to update/extend https://github.com/norwnd/solana/blob/program-sign-only/docs/src/cli/deploy-a-program.md#redeploy-a-program section to document `solana program upgrade` as "preferred" upgrade method ?